monitoring-plugins/check_openshift.py

```diff
@@ -67,10 +69,26 @@
                     help='Check status of all nodes')
 PARSER.add_argument("--check_pods", action='store_true',
                     help='Check status of pods ose-haproxy-router and ose-docker-registry')
+PARSER.add_argument("--exclude_pods", type=str,
+                    help='Exclude pods where the name contains the string passed in argument, find in the field deploymentconfig labels str|str|str ')
```

Comment:
+1 fridim Could be nice.

@axil76 regarding the list, here a example: https://github.com/talset/ssl-tools/blob/master/ssltools.py#L44